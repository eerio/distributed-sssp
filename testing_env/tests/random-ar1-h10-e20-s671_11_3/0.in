11 0 3
0 1 18
0 4 17
0 5 17
0 8 1
1 0 18
1 2 4
1 10 10
2 1 4
2 3 12
3 2 12
3 4 14
3 8 2
