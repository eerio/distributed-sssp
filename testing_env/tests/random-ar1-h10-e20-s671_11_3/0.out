0
14
15
3